### DISCUSSION TITLE
LosslessCut won't start on Windows. No error message... Just nothing.

### DISCUSSION THREAD

**Discussion by TechManiacHD** [mifi/lossless-cut#2043] [Q&A] [answered]
Hi guys!

My LosslessCut that has been bought via Windows Store doesn't work. Click and simply... Not starting. There is no message with error, not a thing. Honestly don't know where to start and how to fix this issue.

It was working perfectly fine. Then I went for a one-month holiday, went back and it's not working. What's more! Asus Armoury Crate has similar problem. Seems like start loading, and then nothing at all... Uninstalling/reinstalling solves nothing. I went to Windows updates to catch up with everything on my Windows 11 Pro, a few restarts, a few days and both, Armoury Crate and LosslessCut won't start. 

Would you have any idea how to solve this? Do those two depend on some SW that has been... IDK, broken by itself? xD

**Comment by mifi** [comment]
Hi. that's very odd. It definitely sounds like something changed on your system causing these apps to stop working all of a sudden.

- Can you open the Event Viewer and see if anything gets logged there?
- Or start LosslessCut.exe from the command prompt and see if any error gets printed (you'd need to find the installed path to LosslessCut then, I think it should be somewhere under `C:\Program Files\WindowsApps\`).
- Another wild shot is to try to run it as administrator.
- check if it could be an anti-virus/firewall causing problems (maybe try to temporarily turn it off)
- Check if Google Chrome has a similar problem (LosslessCut is based on Electron/Chromium)
- Check if other electron-based apps has the same problem (Slack/Discord/MS Teams/Skype)

**Comment by TechManiacHD** [comment]
Thank you for the prompt response.

- The Event Viewer doesn't show any new issues when attempting to start LosslessCut.
- I skipped it.
- I tried running it as an administrator, but to no avail.
- I disabled AdGuard and Bitdefender, yet the issue persists.
- Chrome is functioning perfectly.
- Discord is also working; I don't use any other applications.

My solution wasn't exactly a fix. I restored a backup that I made about 50 days ago. It seems to be working so far. I am in the process of updating Windows. Should the issue recur, I will inform you. Otherwise, I don't know what caused the problem.

**Comment by TempusFugit1353** [comment]
Is it possible that you extracted the compressed file and then moved the files to another location?  I accidentally did this and failed to copy the locales and resources folders and then encountered a problem exactly as you describe.  I extracted the package again into its own folder and it launched fine then.

**Comment by PA1pi4** [comment, marked as answer]
I encountered a similar problem when I tried to download LosslessCut to my Windows 11 laptop (I had previously used LosslessCut on my main Windows 10 computer, and it worked fine). On the laptop, the program ran smoothly only if I launched it from the directory where it was unzipped. But as soon as I dragged the program files to any other directory, after launching the .exe file, only a blank program window (without a user interface) appeared, which closed after 10 seconds. <img width="990" height="811" alt="Снимок экрана 2025-11-08 135611" src="https://github.com/user-attachments/assets/292a390d-0717-4ef5-b850-4e05623029e7" /> 
I tried reinstalling different versions of the program, but the problem persisted. I found two solutions:
1. If you run the program's .exe file from the command line, you may receive error messages.
<img width="1454" height="628" alt="Снимок экрана 2025-11-08 140233" src="https://github.com/user-attachments/assets/5a3dfe72-b99c-4ef8-9cbc-e4c6e18f0d76" /> However, if I run it with the arguments --disable-gpu-sandbox --disable-features=VizDisplayCompositor, the program launched successfully. 
<img width="1230" height="375" alt="Снимок экрана 2025-11-08 141212" src="https://github.com/user-attachments/assets/ce88fb1e-4487-4219-a2af-4489be9b57ce" /> You can then create a shortcut to the .exe file and specify the previously mentioned parameters in its launch parameters.
2. Similar to the previous solution, navigate to the program directory via the command line, then run three commands in sequence:
icacls "(your path to the program folder without brackets)\*" /reset /T
icacls "(your path to the program folder without brackets)\*" /grant:r "%USERNAME%:(F)" /T
icacls "(your path to the program folder without brackets)" /inheritance:r
The first command removes all AppContainer restrictions (which, apparently, are causing the errors) and restores inherited permissions from the parent folder.
The second command gives your user full control over all program files.
The third command prevents Windows from automatically applying AppContainer restrictions to this folder in the future.
After running these commands, my LosslessCut launched correctly.
